core/parser.mly

```diff
@@ -899,6 +901,31 @@ database_expression:
                                                                  pos() }
 | DATABASE atomic_expression perhaps_db_driver                 { `DatabaseLit ($2, $3), pos() }
 
+fn_dep_cols: 
```

Comment:
Have you checked that this has zero shift/reduce conflicts?

Reply:
Will try double checking this.

Reply:
Perhaps `ocamlyacc` could be invoked with `--strict` so that conflicts cause a build failure.

Reply:
@yallop Good point! Thanks.

Reply:
@rudihorn is this checked/resolved?

Reply:
`parser.mly` has been fixed and passes with the `--strict` tag.